docker-compose.yml

```diff
@@ -6,28 +6,25 @@ services:
     restart: unless-stopped
     ports:
       - "5212:5212"
+      - "6800:6800"
+      - "6880:6880"
     volumes:
-      - temp_data:/data
-      - ./cloudreve/uploads:/cloudreve/uploads
-      - ./cloudreve/conf.ini:/cloudreve/conf.ini
-      - ./cloudreve/cloudreve.db:/cloudreve/cloudreve.db
-      - ./cloudreve/avatar:/cloudreve/avatar
-    depends_on:
-      - aria2
+      - ./data:/data
+      - temp_data:/tmp
   aria2:
     container_name: aria2
     image: p3terx/aria2-pro # third party image, please keep notice what you are doing
     restart: unless-stopped
+    network_mode: service:cloudreve
     environment:
-      - RPC_SECRET=your_aria_rpc_token # aria rpc token, customize your own
-      - RPC_PORT=6800
+      - RPC_SECRET=your_aria_rpc_token # 请务必修改此处 Token 为随机字符串, 否则其他人都可以使用你的 Aria2 服务器
     volumes:
-      - ./aria2/config:/config
-      - temp_data:/data
+      - ./aria2_config:/config
+      - temp_data:/tmp
+  aria2-ui:
```

Comment:
no need, aria2 should be part of module list of Cloudreve, we don't provide ui dashboard for docker image, actually the remote download page inside Cloudreve frontend is the dashboard(ui), if user need customize download option, I think using another aria2 image with ui would be better, do not include different download option in the Cloudreve image.